config.toml

```diff
@@ -0,0 +1,25 @@
+ignoreGeneratedHeader = false
```

Comment:
Is it possible to name this `revive_config.toml` just to prevent any confusion about what this file is for?

Reply:
@techknowlogick OK, do it.

Reply:
@lafriks @techknowlogick change file name to `.revive.toml`.